config/profile/backstage.io/manager.yaml

```diff
@@ -71,24 +71,32 @@ spec:
       - command:
         - /manager
         args:
-        - --leader-elect
+          - --health-probe-bind-address=:8081
+          - --metrics-bind-address=:8443
+          - --metrics-secure=true
+          - --leader-elect
         image: controller:latest
         name: manager
         securityContext:
           allowPrivilegeEscalation: false
           capabilities:
             drop:
               - "ALL"
+        ports:
+          - name: health
+            containerPort: 8081
+          - name: metrics
+            containerPort: 8443
         livenessProbe:
           httpGet:
             path: /healthz
-            port: 8081
+            port: health
```

Comment:
Since this alias is used for both health and rediness I'd suggest to rename it to something like "probe" instead

Reply:
IMO, probe is too generic as a term, since it might have different meaning (like a probe for scanning the network, or for collecting logs, or even for providing remote control...).
The liveness and readiness probes defined here both aim at providing health checks on that same port. So "health" as port name would be a more meaningful alias IMO. Or maybe "health-probe" if you prefer?
I think keeping "health" in the name helps better understand the purpose of this port.

Reply:
To me "health" and "readiness" have different meaning (it may be health but not ready for ex) but both are "probes".
Another option would be "liveness"

Reply:
> To me "health" and "readiness" have different meaning
> [...]
> Another option would be "liveness"
>
Not sure I get your point. We are talking about "liveness" and "readiness" probes, right?
And both are targetting the same port alias: [liveness](https://github.com/rm3l/redhat-developer-hub-operator/blob/RHIDP-4236--remove-kube-rbac-proxy-and-use-WithAuthenticationAndAuthorization-instead/config/profile/backstage.io/manager.yaml#L90-L93) / [readiness](https://github.com/rm3l/redhat-developer-hub-operator/blob/RHIDP-4236--remove-kube-rbac-proxy-and-use-WithAuthenticationAndAuthorization-instead/config/profile/backstage.io/manager.yaml#L96-L99)
Naming the port alias "liveness" would make it look like it is specific to the liveness probe and make it different from the port used by the readiness probe..